src/pages/ESNMembersTab/EsnersSheet.tsx

```diff
@@ -15,12 +20,26 @@ const createEsner = (rawEntry: string[]) => {
         id: parseInt(rawEntry[0]),
         name: rawEntry[1],
         surname: rawEntry[2],
-        isBoard: Boolean(rawEntry[3]),
+        memberType: rawEntry[3],
         position: rawEntry[4],
         picture: rawEntry[5],
     };
 };
 
+function showMembers(data: ESNer) {
+    return (
+        <IonItem key={data.id} button onClick={() => alert('Wybrałeś ' + data.surname)}>
+            <img alt="ESN Member photo" className="member-thumbnail" width="60" height="60" src={data.picture} />
+            <p>
+                <strong>
+                    {data.name} {data.surname}{' '}
```

Comment:
why `{' '}` ? 

If we can live without it (eg. it was added by formatter etc) , please remove it ;)